Packages/com.unity.inputsystem/InputSystem/Editor/AssetImporter/InputActionCodeGenerator.cs

```diff
@@ -130,72 +157,25 @@ public static string GenerateWrapperCode(IEnumerable<InputActionMap> maps, IEnum
                     }
                 }
             }
-            writer.WriteLine("m_Initialized = true;");
             writer.EndBlock();
 
-            // Uninitialize method.
-            writer.WriteLine("private void Uninitialize()");
-            writer.BeginBlock();
-            foreach (var map in maps)
+            writer.WriteLine("public void Enable()");
+			writer.BeginBlock();
+            foreach (var set in maps)
             {
-                var mapName = CSharpCodeHelpers.MakeIdentifier(map.name);
-
-                if (options.generateInterfaces)
```

Comment:
But this path is _uninitialization_ of interfaces (`SetCallbacks(null)`). The old code would call this when changing the asset attached to the wrapper. When would we need or do uninitialization today?

Reply:
Also, I wonder: is there a reason to ever not generate this? Ie, should we remove `options.generateInterfaces` and always generate them? After all, just because we generate the code doesn't mean you have to use it.

Reply:
>But this path is uninitialization of interfaces (SetCallbacks(null)). The old code would call this when changing the asset attached to the wrapper. When would we need or do uninitialization today?

Ah right, I read the diff incorrectly. Didn't realize the rest of the codepaths were still there.

Resetting the callbacks may still be useful in some situations but certainly doesn't seem to be a reason to do that automatically.

>is there a reason to ever not generate this? Ie, should we remove options.generateInterfaces and always generate them? After all, just because we generate the code doesn't mean you have to use it.

True. I think it's fine to do that. The current switches came more out of an experimentation with different approaches. generateInterfaces proved to be one of the more successful ones.